### DISCUSSION TITLE
How do I use tags?

### DISCUSSION THREAD

**Discussion by sirlaughalot90** [mcndt/obsidian-toggl-integration#22] [Q&A]
Hello,

just a quick question: How to I apply tags? Thank you!

**Comment by mcndt** [comment]
Hey! Thanks for writing. 

I currently don't support applying tags, though if you restart a past time entry that had tags they should be applied on the new timer.

Someone else has already started a thread on discussing the implementation of tags, so I would be very grateful if you could leave some feedback on this thread! see #20 

Once I know how people like to use tags I will implement it in the plugin.
